app/src/main/java/com/eventyay/organizer/core/event/create/EventDetailsStepThree.java

```diff
@@ -134,7 +134,10 @@ private void setOriginalImageUrl(ImageUrl imageUrl) {
 
     @Override
     public void showError(String error) {
-        Toast.makeText(getActivity(), error, Toast.LENGTH_SHORT).show();
+        if(!error.equals("")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/app/fossasia/open-event-orga-app/pullRequest?prid=4568976)